src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -1,23 +1,595 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-/**
- * [[ClientMetricReport]] gets the media metrics used by ConnectionMonitor to
```

Comment:
Can we keep the comment for interface?

Reply:
Added for ClientMetricReport and StatsCollector.